Source/ModuleKISInventory.cs

```diff
@@ -1301,8 +1309,10 @@ public enum InventoryType {
     GUILayout.BeginHorizontal();
 
     GUILayout.BeginVertical();
-    const int Width = 160;
-    GUILayout.Box("", GUILayout.Width(Width), GUILayout.Height(100));
+            
+	int Width = (int)Math.Round(160.0 * GameSettings.UI_SCALE * GameSettings.UI_SCALE_APPS);
```

Comment:
Looks like you've put tab symbols here. Please, fix to spaces and check the indentation.

Reply:
I thought I had fixed all the indentation. While I was working on this made the mistake of hitting CTRL-K-CTRL-D in Visual Studio that reformats the entire document according to the style defined in visual studio. I didn't really notice until I compared my changes against the original and had every line flagged as different. I spent quite a bit of time cleaning up that mistake.